db/seeds.rb

```diff
@@ -119,4 +119,46 @@
   counting_page.parent_id = ContentPage.find_by(title: "Maths").id
   counting_page.markdown = markdown_for_counting
   counting_page.save!
+
```

Comment:
Please don't add to the seeds.rb,  we changed it with real content today, so this is not needed

Reply:
this PR predates the merged seeds PR thats why it's in here. But i've been able to merge main to this and remove the seeds i needed at add at the time..